io.cryobyte.cryobyteutilities.appdata.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.cryobyte.cryobyteutilities</id>
+  
+  <name>Cryobyte Utilities</name>
+  <summary>A tool used enhancing video game performance on Linux. Designed primarily for the Steam Deck, but is usable on multiple Linux Distibutions.</summary>
+
+  <developer_name>Cryobyte33</developer_name>
+  
+  <metadata_license>MIT</metadata_license>
```

Comment:
Per https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-metadata_license, the metadata license supported does not appear to include GPL. This is not the code/project license, this is strictly the license for this metadata itself. Other options are:

FSFAP
MIT
0BSD
CC0-1.0
CC-BY-3.0
CC-BY-4.0
CC-BY-SA-3.0
CC-BY-SA-4.0
GFDL-1.1
GFDL-1.2
GFDL-1.3
BSL-1.0
FTL
FSFUL

@CryoByte33 Do you have a preference on this, or are you comfortable with MIT?